jgiven-core/src/main/java/com/tngtech/jgiven/impl/util/AnsiUtil.java

```diff
@@ -1,13 +1,13 @@
 /**
  * Copyright (C) 2009, Progress Software Corporation and/or its
  * subsidiaries or affiliates. All rights reserved.
- *
+ * <p>
```

Comment:
Uh, what's the point of this? :)

Reply:
checkstyle likes it better that way, I guess...